### DISCUSSION TITLE
quasar dev command

### DISCUSSION THREAD

**Discussion by nampewo** [quasarframework/quasar#16175] [CLI - BEX mode]
why do i get App • ⚠️  Failed to open default browser whenever i use quasar dev

**Comment by dongwa** [comment]
Do you add this to your quasar.config.js?

![image](https://github.com/quasarframework/quasar/assets/38457491/898998aa-29a3-4c06-a5c9-fcc02541622d)

**Reply by Pinparker** [reply]
Yes, but no
